src/mantine-hooks/src/use-network/use-network.ts

```diff
@@ -47,7 +44,7 @@ export function useNetwork() {
   useWindowEvent('offline', () => setStatus({ online: false, ...getConnection() }));
 
   useEffect(() => {
-    if (navigator.connection) {
+    if (navigator?.connection) {
```

Comment:
In which case navigator can be undefined?

Reply:
I'm not sure what's the status in SSR. I've added it to be on the safe-side.
Also, there is `if (typeof navigator === 'undefined') {` in `getConnection` so I am keeping it consistent.

Reply:
useEffect hooks are never called during ssr, while `getConnection` function will be called since it is used as an initial value for useState

Reply:
Got it. You want the check removed?

Reply:
Yep, seems like ut is not required